assets/pages/contributions-landing/components/contributionsBundle.jsx

```diff
@@ -102,21 +105,31 @@ function showPayPalError(props: PropTypes) {
 }
 
 const ctaLinks = {
-  recurring: routes.recurringContribCheckout,
+  annual: routes.recurringContribCheckout,
+  monthly: routes.recurringContribCheckout,
   oneOff: routes.oneOffContribCheckout,
 };
 
 
 // ----- Functions ----- //
 
+function getContribKey(contribType) {
+  switch (contribType) {
+    case 'ANNUAL': return 'annual';
+    case 'MONTHLY': return 'monthly';
+    default: return 'oneOff';
+  }
+}
```

Comment:
Is this the same as [this](https://github.com/guardian/support-frontend/pull/211/files#diff-fe6c24c6516d4c9d54254e46434d7d1dR160)?

Reply:
Yes, I couldn't see a good place to put it for reuse so I duplicated it as it's small and only used twice. If you have a suggestion for where it might go I'm happy to refactor?

Reply:
It's an irritating problem to have, having to translate an identical word back and forth between different casings. I guess the only place I can think of is `contributions.js`...